### PR TITLE
[FIX] portal_sale_distributor: Adjust to avoid bugs with other values in context

### PR DESCRIPTION
**Pull request by nicomacr**

### CODE REVIEW

**Comment by roboadhoc** [comment]
[Pull request status dashboard](https://runbot.adhoc.com.ar/ingadhoc/sale/pull/492).

File: portal_sale_distributor/models/ir_action_act_window.py [hunk 1]
```diff
@@ -11,10 +11,10 @@ class ActWindowView(models.Model):
 
     def read(self, fields=None, load='_classic_read'):
         result = super().read(fields, load=load)
-        if result and result[0].get('context'):
-            ctx = safe_eval(result[0].get('context', '{}'))
-            if ctx.get('portal_products'):
+        for value in result:
+            if value.get('context') and 'portal_products' in value.get('context'):
+                ctx = safe_eval(value.get('context', '{}'))
```
Comment (jjscarafia) [inline portal_sale_distributor/models/ir_action_act_window.py]: cambiara por
                    eval_ctx = dict(self.env.context)
                    try:
                        ctx = safe_eval(values.get('context', '{}'), eval_ctx)
                    except:
                        ctx = {}

File: portal_sale_distributor/models/ir_action_act_window.py [hunk 2]
```diff
@@ -11,10 +11,10 @@ class ActWindowView(models.Model):
...
                 pricelist = self.env.user.partner_id.property_product_pricelist
                 ctx.update({'pricelist': pricelist.id, 'partner': self.env.user.partner_id})
-                result[0].update({'context': ctx})
+                value.update({'context': ctx})
```
Comment (jjscarafia) [inline portal_sale_distributor/models/ir_action_act_window.py]: value.update({'context': str(ctx)})

**Review by jjscarafia — CHANGES_REQUESTED** [review]

**Comment by jjscarafia** [comment]
@roboadhoc r+

**Pull request closed without merging**